src/App.tsx

```diff
@@ -74,7 +74,9 @@ const App = () => {
       if (localStorage.token && !isTokenAnonymous() && !isLoggedInTokenExpired() && !user.id) {
         getUserData()
           .then((response) => {
-            dispatch(setUser(response.data));
+            getUserAuthorizationsInstitution().then((autResponse) =>
```

Comment:
vi kunne kanskje gjort om disse to kallene til async/await her også ?

Reply:
Fikset